Arelle/Arelle
# Create config for Read the Docs project

#### Reason for change
Closes #785

#### Description of change
* Adds `requirements-docs.txt` for dependencies specific to building docs
* Implements barebones documentation landing page with markdown formatting, placeholder logo and styling
  * Uses [Furo](https://pradyunsg.me/furo/quickstart/) theme 
* Adds `.readthedocs.yml` which RTD uses to build and host documentation. Example: https://aaroncameron-arelle.readthedocs.io/en/latest/index.html

#### Steps to Test
* Build docs locally, see `build_docs.md`
* Confirm above hosted docs appear as expected (optionally fork this branch and publish your own to RTD)

![Screenshot 2023-08-02 at 1 18 24 PM](https://github.com/Arelle/Arelle/assets/105066394/f69f3a82-9ede-4fca-a242-81827b788403)

![Screenshot 2023-08-02 at 1 18 45 PM](https://github.com/Arelle/Arelle/assets/105066394/d0461b28-2118-49f5-ae09-b4edc4a03a52)


**review**:
@Arelle/arelle
